packages/calcite-components/src/components/table-cell/table-cell.scss

```diff
@@ -42,44 +54,67 @@ td.last-cell {
   border-inline-end: 0;
 }
 
-.number-cell {
-  @apply bg-foreground-2;
-}
-
 .footer-cell {
-  @apply bg-background font-medium text-color-1;
-  border-block-start: 1px solid var(--calcite-color-border-3);
+  @apply font-medium text-color-1;
```

Comment:
should this "text-color-1" have a CSS var/token?

Reply:
Yep, will update